examples/configs/neurobooth_os_config.json

```diff
@@ -1,26 +1,28 @@
 {
-    "data_out": "C:\\neurobooth\\neurobooth_data\\\\",
-    "nas": "Z:\\data\\\\",
-    "video_tasks": "C:\\Users\\STM\\Dropbox (Partners HealthCare)\\Neurobooth Videos for tasks\\Videos_to_present",
+    "remote_data_dir": "C:/data/",
```

Comment:
This is supposed to be NAS, right? -> Z:/ instead of C:/

Reply:
Since this is an example file, I thought the actual data was immaterial. And I don't have a Z drive for testing. Or a D drive

Reply:
Missed that intent - makes sense!